HowTo/JBOSS/avoid_timeouts_obsolete_nodes.md

```diff
@@ -0,0 +1,29 @@
+# How to: Avoid JBOSS clustering timeouts due to obsolete discovery files
+
+During the normal lifecycle of your JBOSS EAP app there can be multiple events that can trigger your app to move across different nodes on the App Service infrastucture, this can be caused by user opertations, such as deployments, scale operations, etc... or platform maintenance oeprations.
+
+Over time this will cause an acumulation of stale discovery files leading to potential timeouts for the cluster.
+
+To avoid this the best practice is to include the cleanup of this stale files as part of the application startup using the JBOSS CLI.

```

Comment:
Over time this will cause an accumulation of stale cluster discovery files leading to potential timeouts during startup. To avoid this scenario, the best practice is to include the cleanup of this stale files as part of the application configuration using the JBoss CLI.